Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileAcceptanceTest.cs

```diff
@@ -168,9 +168,9 @@ public void TestExists()
 				new Action(() => logFile = new TextLogFile(_scheduler, File2Lines)).ShouldNotThrow();
 
 				logFile.Property(x => x.EndOfSourceReached).ShouldEventually().BeTrue(TimeSpan.FromSeconds(5));
-				logFile.Property(x => x.GetValue(LogFileProperties.EmptyReason)).ShouldEventually().Be(ErrorFlags.None, TimeSpan.FromSeconds(5));
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.